src/main.js

```diff
@@ -189,27 +193,43 @@ function goBackToMainSaved(){
 
 //Iteration 2
 function showUserPoster(){
-  // event.preventDefault()
-  // var newUserImage = inputFieldForImage.value;
-  // var newUserTitle = inputFieldForTitle.value;
-  // var newUserQuote = inputFieldForQuote.value;
-  // var newUserPoster = new Poster(newUserImage, newUserTitle, newUserQuote);
-  // image.src = newUserImage;
-  // title.innerText = newUserTitle;
-  // quote.innerText = newUserQuote;
-  // images.push(newUserImage);
-  // titles.push(newUserTitle);
-  // quotes.push(newUserQuote);
-  // goBackToMainMake();
+  event.preventDefault()
+  var newUserImage = inputFieldForImage.value;
+  var newUserTitle = inputFieldForTitle.value;
+  var newUserQuote = inputFieldForQuote.value;
+  var newUserPoster = new Poster(newUserImage, newUserTitle, newUserQuote);
+  image.src = newUserImage;
+  title.innerText = newUserTitle;
+  quote.innerText = newUserQuote;
+  images.push(newUserImage);
+  titles.push(newUserTitle);
+  quotes.push(newUserQuote);
+  goBackToMainMake();
 }
 // Iteration 3
-function pushUserPosterToArray(){
+function saveUserPoster(){
+  var newUserImage = inputFieldForImage.value;
+  var newUserTitle = inputFieldForTitle.value;
+  var newUserQuote = inputFieldForQuote.value;
+  var newUserPoster = new Poster(newUserImage, newUserTitle, newUserQuote);
+
+  savedPosters.push({newUserPoster})
   // create variables for user input fields query selectors
   // push variables into arrays
   // push new poster to saved posters arrays by calling the showUserPoster function?
   // create new object to push into the saved posters arrays
   //
 }
+
+function displayUserPoster(){
```

Comment:
where will this be invoked?
an event listener? this function looks a lot like the showUserPoster.

...but this is a draft and we can definitely refactor later an when we get through the iterations.